### PR TITLE
Regenerated Websites SDK code from latest Swagger spec

### PR DESCRIPTION
**Pull request by nking-1**
This commit updates the Websites SDK to match the latest API.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nking92__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nicking). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/DatabaseServerType.cs [hunk 1]
```diff
@@ -13,18 +13,14 @@ namespace Microsoft.Azure.Management.WebSites.Models
     using System.Runtime.Serialization;
 
     /// <summary>
-    /// Defines values for DatabaseServerType.
+    /// Defines values for CertificateProductType.
     /// </summary>
     [JsonConverter(typeof(StringEnumConverter))]
-    public enum DatabaseServerType
```
Comment (shahabhijeet) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/DatabaseServerType.cs]: >DatabaseServerType [](start = 16, length = 18)

Breaking change? would need a major version bump

Comment (Nking92) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/DatabaseServerType.cs]: That's true, we will have to bump the version.

File: src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/ConnStringInfo.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ public partial class ConnStringInfo
         /// <summary>
         /// Initializes a new instance of the ConnStringInfo class.
         /// </summary>
-        public ConnStringInfo(DatabaseServerType? type, string name = default(string), string connectionString = default(string))
```
Comment (shahabhijeet) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/ConnStringInfo.cs]: >DatabaseServerType [](start = 30, length = 18)

breaking change, major version needs to be updated?

File: src/ResourceManagement/WebSite/WebSite.Tests/ScenarioTests/WebSite.ScenarioTests.cs [hunk 3]
```diff
@@ -218,7 +218,7 @@ public void GetAndSetSensitiveSiteConfigs()
                     var connStringValueTypePair = new ConnStringValueTypePair
                     {
                         Value = connectionStringValue,
-                        Type = DatabaseServerType.MySql
```
Comment (shahabhijeet) [inline src/ResourceManagement/WebSite/WebSite.Tests/ScenarioTests/WebSite.ScenarioTests.cs]: >DatabaseServerType [](start = 31, length = 18)

I don't see any new test for the new functionality that was added. Can you add test coverage for the new functionality.

**Comment by shahabhijeet** [comment]
Need test coverage and also major version update for the breaking change that was introduced.

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

**Comment by shahabhijeet** [comment]
@Nking92 will be closing this PR by end of this week if it continues to fail CI

**Comment by markcowl** [comment]
@Nking92 Closing the P{R as indicated.  Please open a new PR once you are ready to merge in your API changes.

**Pull request closed without merging**